### PR TITLE
Reg pack/internal password reset fork

### PR DESCRIPTION
**Pull request by timtamimi**
This PR is a fork of the current regression packs, modified in such a way that they can now pass when run through a GitHub worker run. 

Last successful run can be found here: https://github.com/DEFRA/water-abstraction-ui/runs/1644886748?check_suite_focus=true

I recommend that `reg-pack/internal-password-reset-fork` is merged into `reg-pack/internal-password-reset`,

... And then `reg-pack/internal-password-reset` can be merged into `develop`.

### CODE REVIEW

File: test/regression/internal/reset-password.spec.js [hunk 1]
```diff
@@ -24,51 +22,73 @@ describe('internal user resetting their password:', function () {
...
-    getButton('Continue').click();
+  it('shows a validation message if the email field is empty', async () => {
+    const email = await $('#email');
+    const continueButton = await getButton('Continue');
+    const validationMessage = await getValidationSummaryMessage();
 
-    expect(getValidationSummaryMessage()).toHaveText('Enter an email address');
+    email.setValue('');
+    continueButton.click();
+    expect(validationMessage).toHaveText('Enter an email address in the right format');
```
Comment (DGajewska) [inline test/regression/internal/reset-password.spec.js:32]: Same as with external password reset. I'm not sure that this validation message is correct

**Review by DGajewska — CHANGES_REQUESTED** [review]

File: test/regression/external/reset-password.spec.js [hunk 2]
```diff
@@ -2,75 +2,97 @@
...
-    getButton('Continue').click();
+  it('shows a validation message if the email field is empty', async () => {
+    const email = await $('#email');
+    const continueButton = await getButton('Continue');
+    const validationMessage = await getValidationSummaryMessage();
 
-    expect(getValidationSummaryMessage()).toHaveText('Enter an email address');
+    email.setValue('');
+    continueButton.click();
+    expect(validationMessage).toHaveText('Enter an email address in the right format');
```
[the remark this answers is not in the document]
Comment (timtamimi) [reply test/regression/external/reset-password.spec.js:36]: You're 100% correct. I will amend this.

File: test/regression/external/reset-password.spec.js [hunk 3]
```diff
@@ -2,75 +2,97 @@
...
 
 const EMAIL_ADDRESS = 'acceptance-test.external@example.com';
 
 /* eslint-disable no-undef */
 describe('external user resetting their password:', function () {
   before(async () => {
-    // registerUser(EMAIL_ADDRESS);
-    await setUp();
-    browser.url(`${config.baseUrl}/signin`);
+    await browser.url(baseUrl);
```
[the remark this answers is not in the document]
Comment (timtamimi) [reply test/regression/external/reset-password.spec.js]: You're right, and that makes me wonder why the tests were passing previously...

File: test/regression/internal/reset-password.spec.js (hunk 1 above)
Comment (timtamimi) [reply test/regression/internal/reset-password.spec.js]: ✅

**Comment by timtamimi** [comment]
Changes applied following PR feedback.

CI: https://github.com/DEFRA/water-abstraction-ui/runs/1649737117?check_suite_focus=true
Regression pack: https://github.com/DEFRA/water-abstraction-ui/runs/1649738252?check_suite_focus=true

**Review by DGajewska — APPROVED** [review]

**Pull request merged**